packages/typespec-test/test/openai_generic/generated/typespec-ts/review/openai-generic.api.md

```diff
@@ -797,10 +797,16 @@ export interface OpenAIFile {
 }
 
 // @public
-export type Prompt = string | string[] | number[] | number[][];
+export type Prompt = Prompt_1 | null;
 
 // @public
-export type Stop = string | string[];
+export type Prompt_1 = string | string[] | number[] | number[][];
```

Comment:
is this because tcgc wrongly tagged this union type as name type but it should be an anoymous one?

Reply:
yeah, see issue here https://github.com/Azure/typespec-azure/issues/2133